fox_news_scraper/clean_fn.py

```diff
@@ -10,14 +10,19 @@
 STOPWORDS = set(stopwords.words("english"))
 
 
+def clean_multiple_spaces(to_clean: str) -> str:
+    """Returns text multiple spaces removed"""
+    return re.sub(r'\s+', ' ', to_clean)
+
+
 def clean_text(text: str) -> str:
     """Cleans the article text by removing unwanted characters, stopwords etc."""
 
     text = re.sub(r'http\S+', '', text)
-    text = re.sub(r'[^\w\s]', '', text)
```

Comment:
Removing the line that filters out punctuation is a good idea, will make for some interesting analysis later if we can chart the difference in sentiment between sentences in quotes and sentences without quotes :)